chucks-pick3-web/angular-app/src/app/services/mock-customer.service/mock-customer.service.ts

```diff
@@ -8,10 +8,12 @@ export class MockCustomerService {
   constructor() { }
   public generateCustomer() {
```

Comment:
Please add a return type in method declaration.

Reply:
Added return type method in declaration.